layouts/docs/list.html

```
@@ -0,0 +1,31 @@
+{{/* Docs section layout using Neo components */}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)